File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/score/ScorePutDTO.java
package es.jocasolo.competitiveeventsapi.dto.score;

import java.io.Serializable;

import es.jocasolo.competitiveeventsapi.dto.DTO;
import es.jocasolo.competitiveeventsapi.enums.score.ScoreStatusType;
import lombok.Getter;
import lombok.Setter;

public class ScorePutDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	@Getter
	@Setter
	private String value;

	@Getter
	@Setter
	private ScoreStatusType status;

	@Getter
	@Setter
	private String eventId;

	@Override
	public String toString() {
		return String.format("ScorePutDTO [value=%s]", value);
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/score/ScorePutDTO.kt
package es.jocasolo.competitiveeventsapp.dto.score

import com.google.gson.annotations.SerializedName
import es.jocasolo.competitiveeventsapp.dto.BackStackEntryDTO
import es.jocasolo.competitiveeventsapp.enums.score.ScoreStatusType

class ScorePutDTO(
        @SerializedName("value") var value: String? = null,
        @SerializedName("status") var status: ScoreStatusType? = null,
        @SerializedName("eventId") var eventId: String? = null,
) : BackStackEntryDTO() {
}